index.html

```diff
@@ -431,7 +440,7 @@
 
       <p>Pointer lock must be exited if the
       <a>target</a> is removed from its document, or the <a>user agent</a>,
```

Comment:
...if the target is *disconnected*, or the ...

look okay?  It seems "disconnected" is not defined.  How about

... if the target is *removed and is no longer in a shadow-including document*, or the ...

?